docs/ja/sql-reference/statements/select/offset.md

```diff
@@ -0,0 +1,86 @@
+---
+toc_title: OFFSET
+---
+
+# OFFSET FETCH Clause {#offset-fetch}
```

Comment:
Why is it `english` doc in `japanese` doc?

Reply:
It was a symbolic link before. We have not Japanese for this text. AFAIK we refuse symbolic links.